lib/components/TextLinkRenderer/TextLinkRenderer.tsx

```diff
@@ -22,34 +24,66 @@ export default class TextLinkRenderer extends Component<TextLinkRendererProps> {
 
     return (
       <ThemeConsumer>
-        {theme => {
-          const defaultStyles = [
-            styles.root,
-            theme.atoms.reset.a,
-            theme.atoms.color.link,
-          ];
-
-          if (inline) {
-            return children({
-              style: {},
-              className: classnames(defaultStyles),
-            });
-          }
-
-          return (
-            <Text baseline={false}>
-              {children({
-                style: {},
-                className: classnames(
-                  defaultStyles,
-                  theme.atoms.display.inlineBlock,
-                  theme.atoms.paddingTop.standardTouchableText,
-                  theme.atoms.paddingBottom.standardTouchableText,
-                ),
-              })}
-            </Text>
-          );
-        }}
+        {({ atoms }) => (
+          <ActionsConsumer>
+            {inActions => {
+              const defaultStyles = [
+                styles.root,
+                atoms.reset.a,
+                atoms.color.link,
+              ];
+
+              if (inline) {
```

Comment:
Thoughts on hoisting this to before the `ActionsConsumer`? If it's `inline` it has no contextual behaviour so can return fast.

Reply:
Ah yeah, nice idea 👍